<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindBid</class>
 <widget class="QWidget" name="FindBid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="6" column="0">
    <widget class="QTableWidget" name="tableWidgetResult">
     <column>
      <property name="text">
       <string>I_ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I_E</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I_STR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I_B</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I_K</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I_ADD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B_E</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B_STR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B_B</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B_K</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B_ADD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B_ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonFind">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonOpenIn">
     <property name="text">
      <string>OpenIn</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButtonClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonOpenBase">
     <property name="text">
      <string>OpenBase</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonStop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
